archive/imo/imo1960_q1.lean

```diff
@@ -0,0 +1,169 @@
+/-
+Copyright (c) 2020 Kevin Lacker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Lacker
+-/
+
+import data.nat.digits
+
+open nat
+
```

Comment:
The linter wants these lines to come after the module docstring.

Reply:
OK - looks like the `open` must come after, and the `import` must come before. Fixed

Reply:
Yep, my bad.